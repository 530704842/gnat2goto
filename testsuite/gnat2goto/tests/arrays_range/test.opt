ALL XFAIL gnat2goto fails with low bound range not literal
