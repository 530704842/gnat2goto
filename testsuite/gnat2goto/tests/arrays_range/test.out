[assertion.1] Range Check: FAILURE
[assertion.2] Range Check: SUCCESS
[assertion.3] Range Check: FAILURE
[precondition_instance.1] memcpy src/dst overlap: SUCCESS
[precondition_instance.2] memcpy source region readable: SUCCESS
[precondition_instance.3] memcpy destination region writeable: SUCCESS
[1] file arrays_range.adb line 9 assertion: SUCCESS
[2] file arrays_range.adb line 11 assertion: SUCCESS
[3] file arrays_range.adb line 13 assertion: SUCCESS
VERIFICATION FAILED
